modules/apps/portal-template/portal-template-react-renderer-impl/src/main/resources/META-INF/resources/render.es.js

```diff
@@ -14,10 +14,16 @@
 
 import {render} from 'frontend-js-react-web';
 
-export default function (renderFunction, renderData, placeholderId) {
+export default function (renderFunction, renderData, placeholderId, props) {
 	const element = document.getElementById(placeholderId);
 
+	console.log(props);
```

Comment:
:)

Reply:
This I'm planning on keeping... looks great for debugging in production! ;)

Reply:
Ended up not even changing this and passing in the merged props.